fluss-flink/fluss-flink-common/src/main/java/com/alibaba/fluss/flink/row/RowConverters.java

```diff
@@ -0,0 +1,172 @@
+/*
+ * Copyright (c) 2025 Alibaba Group Holding Ltd.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.alibaba.fluss.flink.row;
+
+import com.alibaba.fluss.row.BinaryString;
+import com.alibaba.fluss.row.GenericRow;
+import com.alibaba.fluss.row.TimestampNtz;
+
+import org.apache.flink.table.types.logical.LogicalType;
+
+import java.lang.reflect.Field;
+import java.lang.reflect.Modifier;
+import java.sql.Date;
+import java.time.Instant;
+import java.time.LocalDateTime;
+import java.time.ZoneId;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+
+/**
+ * Utility class for converting between different row representations in Fluss.
+ *
+ * <p>It's primarily used for testing and data conversion operations when interacting with the Fluss
+ * storage system.
+ */
+public class RowConverters {
+    /**
+     * Converts a POJO to a Fluss GenericRow. Special handling is provided for timestamp types.
+     *
+     * @param pojo The POJO to convert
+     * @return A GenericRow containing the POJO's field values
+     * @throws IllegalArgumentException if there's an error accessing the fields
+     */
+    public static GenericRow pojoToGenericRow(Object pojo) {
```

Comment:
I just merged https://github.com/alibaba/fluss/issues/723 which already provides this util. We can remove this method.